src/bika/lims/browser/worksheet/views/referencesamples.py

```diff
@@ -249,7 +249,7 @@ def folderitem(self, obj, item, index):
 
         # Supported Services
         supported_services_choices = self.make_supported_services_choices(obj)
-        item["choices"]["SupportedServices"] = supported_services_choices
+        item["choices"]["SupportedServices"] = sorted(supported_services_choices, key=lambda d: d['ResultText'])
```

Comment:
Better do the sorting inside `make_supported_services_choices` function: https://github.com/senaite/senaite.core/pull/2517/files#diff-315b812041dbdb95c7ba93d6d8b05774cef405c2ff143855eb92882f9034647fL201

Reply:
> Better do the sorting inside `make_supported_services_choices` function: https://github.com/senaite/senaite.core/pull/2517/files#diff-315b812041dbdb95c7ba93d6d8b05774cef405c2ff143855eb92882f9034647fL201

done )

that's bit unobvious why its better, my consideration was that so far the sort makes sense for presentation only I put as close to view layer as possible. I'm just curios why.

Reply:
`supported_services_choices` is in charge of generating the choices for selection (and only meaningful for presentation purposes), so one expects these choices to come sorted by default.